### DISCUSSION TITLE
Mutli-edit rows [May 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2200] [Discord Threads]
> 🙋‍♂️ **johndoe**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### not at the moment  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/977410241413120020)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
